assets/src/edit-story/components/header/buttons/update.js

```diff
@@ -99,28 +92,13 @@ function Update() {
       );
   }
 
-  const tooltip = showPrePublishTab
-    ? checklist.some(({ type }) => PRE_PUBLISH_MESSAGE_TYPES.ERROR === type) &&
-      __('There are items in the checklist to resolve', 'web-stories')
-    : null;
-
-  const button = (
-    <Primary
-      onPointerEnter={refreshChecklist}
+  return (
+    <ButtonWithPrepublishChecklistWarning
```

Comment:
This is a very explicitly named, yet quite a long component & file name.

Could we perhaps use something shorter like `ButtonWithWarning`?

Or maybe `Primary` / `prrimary.js` because it's the primary button in this section?

Reply:
I had a hard time naming this. The one I was working with originally was `ButtonWithChecklistWarning` but didn't seem specific enough. That (preferably) or `ButtonWithWarning` like you suggest would be good.

Reply:
I named it `ButtonWithChecklistWarning`